### PR TITLE
ROC-3575 Enable ccd

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

File: src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RequestCountyCourtJudgementTest.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public void shouldBeAbleToSuccessfullyRequestCCJ() {
 
         assertThat(updatedCase.getCountyCourtJudgment()).isEqualTo(ccj);
         assertThat(updatedCase.getCountyCourtJudgmentRequestedAt())
-            .isCloseTo(LocalDateTimeFactory.nowInLocalZone(), within(2, ChronoUnit.MINUTES));
+            .isCloseTo(LocalDateTimeFactory.nowInUTC(), within(2, ChronoUnit.MINUTES));
```
Comment (timja-kainos) [inline src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RequestCountyCourtJudgementTest.java]: pretty sure this is wrong, we return the local zone back from the api

File: Dockerfile [hunk 2]
```diff
@@ -8,4 +8,4 @@ HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" wget -q
 
 EXPOSE 4400
 
-ENTRYPOINT ["/usr/bin/java", "-jar", "/opt/app/claim-store.jar"]
+ENTRYPOINT exec java ${JAVA_OPTS} -jar "/opt/app/claim-store.jar"
```
Comment (timja-kainos) [inline Dockerfile]: you need to also update hub.Dockerfile

File: src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RequestCountyCourtJudgementTest.java (hunk 1 above)
Comment (timja-kainos) [reply src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RequestCountyCourtJudgementTest.java]: or some reason we stop timezone information but keep it in the db....
"simpler"

All PDFs receive local british time
The API returns british time

File: src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/SettlementOfferTest.java [hunk 3]
```diff
@@ -192,7 +192,7 @@ public void shouldBeAbleToSuccessfullyCountersignOffer() {
         assertThat(caseWithCounterSign.getSettlement().get().getPartyStatements().size()).isEqualTo(3);
 
         assertThat(caseWithCounterSign.getSettlementReachedAt())
-            .isCloseTo(LocalDateTimeFactory.nowInLocalZone(), within(2, ChronoUnit.MINUTES));
```
Comment (yarpo) [inline src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/SettlementOfferTest.java]: what was reason of these updates?

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -38,6 +38,12 @@ idam:
     url: "http://localhost:4552"
     totp_secret: "AAAAAAAAAAAAAAAA"
     microservice: "cmc_claim_store"
+  anonymous-caseworker:
+    username: "${IDAM_ANONYMOUS_CASEWORKER_USERNAME}"
```
[the remark this answers is not in the document]
Comment (dharmendrak) [reply src/main/resources/application.yml]: property with - in the name is not supported by bash or zsh, can't be declared locally.

File: src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RequestCountyCourtJudgementTest.java (hunk 1 above)
Comment (dharmendrak) [reply src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RequestCountyCourtJudgementTest.java]: changed back

File: Dockerfile (hunk 2 above)
Comment (dharmendrak) [reply Dockerfile]: added

File: src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/SettlementOfferTest.java (hunk 3 above)
Comment (dharmendrak) [reply src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/SettlementOfferTest.java]: reverted

File: src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RespondToClaimTest.java [hunk 5]
```diff
@@ -63,7 +63,7 @@ private void shouldBeAbleToSuccessfullySubmit(Response response) {
 
         assertThat(updatedCase.getResponse().isPresent()).isTrue();
         assertThat(updatedCase.getResponse().get()).isEqualTo(response);
-        assertThat(updatedCase.getRespondedAt()).isCloseTo(LocalDateTimeFactory.nowInLocalZone(),
+        assertThat(updatedCase.getRespondedAt()).isCloseTo(LocalDateTimeFactory.nowInUTC(),
```
Comment (timja-kainos) [inline src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RespondToClaimTest.java]: this should be changed back too

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java [hunk 6]
```diff
@@ -121,7 +121,7 @@ public CaseDetails saveCountyCourtJudgment(
 
         CCDCase ccdCase = this.caseMapper.to(claim);
         ccdCase.setCountyCourtJudgment(countyCourtJudgmentMapper.to(countyCourtJudgment));
-        ccdCase.setCountyCourtJudgmentRequestedAt(now());
+        ccdCase.setCountyCourtJudgmentRequestedAt(LocalDateTimeFactory.nowInLocalZone());
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java]: could you static import `LocalDateTimeFactory`

**Pull request merged**